po/de.po

```diff
@@ -22557,7 +22552,7 @@ msgstr "Änderung in der Anzahl von Worttrennern ignorieren"
 
 #: /tmp/fish.i8YroE/implicit/share/completions/diff.fish:6
 msgid "Ignore all white space"
-msgstr "white space (Worttrenner) ignorieren"
+msgstr "Alle Leerzeichen (Worttrenner) ignorieren"
```

Comment:
"Leerzeichen" would specifically be the space character - this is also about tab and such.

Reply:
"Leerräume"?

Reply:
"Leerräume"?